frontends/ast/ast.cc

```diff
@@ -275,12 +277,38 @@ void AstNode::delete_children()
 	attributes.clear();
 }
 
+AstNode* AstNode::find_child(const std::string& name)
+{
+	for (auto child : children) {
+		if (child->str == name) {
+			return child;
+		}
+	}
+	return nullptr;
+}
+
+AstNode* AstNode::find_child(AstNodeType type, const std::string& name)
+{
+	auto child = find_child(name);
+	if (child && child->type == type) {
+		return child;
+	}
+	return nullptr;
+}
+
```

Comment:
Remove, this change is not necessary for multiranges support.

Reply:
Actually, I'm using it in the parser to find a specific type.
Maybe can I move this to another commit?